Sources/SwiftProtobuf/StringUtils.swift

```diff
@@ -50,21 +62,27 @@ internal func utf8ToString(bytes: UnsafePointer<UInt8>, count: Int) -> String? {
     if count == 0 {
         return String()
     }
-#if os(Linux)
-    // As of March, 2017, the NSString(bytes:length:encoding:)
-    // initializer incorrectly stops at the first zero character for
-    // Linux versions of Swift:
+#if !os(Linux) || swift(>=3.2)
```

Comment:
I really wanted `>3.1.1` but that doesn't work.

Has there been any announcement about the next version?  If you know the next version will be 4.0, we could use that here.